kdlc/KDLLoader.py

```diff
@@ -22,12 +24,14 @@ def exitNode_settings(
         self: KDLListener, ctx: KDLParser.Node_settingsContext
     ) -> None:
         node_number = ctx.node().node_id().getText()
-        # print(f"nodeNumber: {node_number}")
+        # logger.debug(f"nodeNumber: {node_number}")
 
         json_tokens = [i.getText() for i in ctx.json().children]
         json_string = "".join(json_tokens)
         node_settings = json.loads(json_string)
 
+        # logger.debug(node_settings)
```

Comment:
Probably good to uncomment the use of logger in here and rip the print statements.

Reply:
Do we need these statements or will it be redundant based on what we have in Commands.py ?

Reply:
Resolved by https://github.com/k-descriptor-language/kdl/commit/e06043de395ebd398dd90572e0dae8ffbc433267